examples/with-supertokens/package.json

```
@@ -3,13 +3,18 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "lint": "next lint"
```

```suggestion
    "start": "next start"
```

Seems to be unnecessary to add linting to this example, we have specific examples for this case.